remote/remote.go

```diff
@@ -53,7 +58,34 @@ func WithPreviousImage(imageName string) ImageOption {
 func FromBaseImage(imageName string) ImageOption {
 	return func(r *Image) (*Image, error) {
 		var err error
-		r.image, err = newV1Image(r.keychain, imageName)
+
+		currentImagePlatform, err := imagePlatform(r.image)
+		if err != nil {
+			return nil, err
+		}
+
+		r.image, err = newV1Image(r.keychain, imageName, currentImagePlatform)
+		if err != nil {
+			return nil, err
+		}
+		return r, nil
+	}
+}
+
+func WithDefaultPlatform(p v1.Platform) ImageOption {
```

Comment:
Right now we try not expose ggcr types directly in the public interface of imgutil (`authn.Keychain` is the convenient exception). I think we should add our own Platform type.

Reply:
If we remove `WithDefaultPlatform` for now, as mentioned above, then we can punt on this issue for the time being.